CMakeLists.txt

```diff
@@ -46,20 +43,12 @@ add_subdirectory("cmake")
 # =============================================================================
 include(PACE_CPackConfig)
 
-if(WIN32)
-    # Don't package external Linux libraries on Windows
-    set(EXTERNAL_IGNORE_PATTERN "*/external/glnxa64")
-endif()
-
 install(
     DIRECTORY "herbert_core"
     DESTINATION "Herbert"
     USE_SOURCE_PERMISSIONS
     PATTERN "*.m~" EXCLUDE
     PATTERN "*.asv" EXCLUDE
-    PATTERN "${EXTERNAL_IGNORE_PATTERN}" EXCLUDE
-)
-install(
-    FILES "LICENSE" "README.md"
-    DESTINATION "."
-)
+    PATTERN "*.gitignore" EXCLUDE
+    PATTERN "${EXTERNAL_IGNORE_PATTERN}" EXCLUDE)
+install(FILES "LICENSE" "README.md" DESTINATION ".")
```

Comment:
Do the README or LICENSE files need updating to note that we are including the MSMPI, MPICH and HDF libraries (with license files in the appropriate `external` directory?

Reply:
What was the decision on this? We discussed offline that we should have a note, did we decide it should be in our licence or in the README? And what should it say exactly?